beard/clustering/tests/test_wrappers.py

```diff
@@ -27,12 +27,25 @@
 from ..wrappers import ScipyHierarchicalClustering
 from beard.metrics import b3_f_score
 
-from sklearn.metrics import silhouette_score
+from beard.metrics import silhouette_score
 from functools import partial
 
 
-# Test wrapper of Scipy hierarchical clustering
+# Defining some scoring functions for related test cases
+def pred_labels_score(pred_labels):
+    """Scoring function that takes 1 exact argument: labels."""
+    return -np.inf
+
+
+def groundTruth_X_score(X, true_labels, pred_labels):
+    """Scoring function that takes 3 exact arguments."""
+    # For the sake of the case test, X is ignored. So, it will be
+    # as using 2 exact arguments function like b3_f_score.
```

Comment:
`as` vs `as if`?
`2 exact arguments function` vs `function that takes exactly 2 arguments`